descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -77,7 +77,7 @@ TEST(CartTrajPt, getPoses)
   std::vector<std::vector<double> >joint_solutions;
 
   ROS_INFO_STREAM("Testing fuzzy pos point");
-  CartesianRobot robot(POS_TOL+2*EPSILON, ORIENT_TOL+2*EPSILON);
+  CartesianRobot robot(POS_TOL+2*EPSILON, M_PI*ORIENT_TOL+2*EPSILON);
```

Comment:
@Jmeyer1292  Would it not be more appropriate to set ORIENT_TOL to a larger value?